public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -194,16 +184,15 @@ include ../_util-fns
 
   In Angular terms, an idempotent expression always returns *exactly the same thing* until
   one of its dependent values changes.
-// #enddocregion template-expressions-guidelines
-// #docregion template-expressions-guidelines-2
 :marked
   Dependent values should not change during a single turn of the event loop.
   If an idempotent expression returns a string or a number, it returns the same string or number
-  when called twice in a row. If the expression returns an object (including a `Date` or `Array`),
+  when called twice in a row. If the expression returns an object (including #{_a} `#{_Array}`),
```

Comment:
Why leave out `Date`?

If there's a good reason to leave it out, change `_a` to `_an`.